gerritlab/merge_request.py

```diff
@@ -132,6 +134,15 @@ def delete(self, delete_source_branch=False):
         if delete_source_branch:
             self._remote.push(refspec=(":{}".format(self._source_branch)))
 
+    def reload(self):
+        """Reloads the MR by closing and reopening it.
+
+        This can be used to refresh MR status.  Related GitLab issue:
+        https://gitlab.com/gitlab-org/gitlab-foss/-/issues/19026#note_36401573
```

Comment:
Would you mind mentioning this in the commit message and the MR overview?

Reply:
Done